# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/dietlibc/pkg_patch/pkg_diffutils.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/cmp.c.orig	Fri Apr  5 22:37:31 2002
+++ ./src/cmp.c	Wed May  1 12:49:39 2002
@@ -91,7 +91,7 @@
 /* Values for long options that do not have single-letter equivalents.  */
 enum
 {
-  HELP_OPTION = CHAR_MAX + 1
+  HELP_OPTION = 256
 };
 
 static struct option const long_options[] =
--- ./src/diff3.c.orig	Fri Apr  5 22:37:31 2002
+++ ./src/diff3.c	Wed May  1 12:49:39 2002
@@ -204,7 +204,7 @@
 /* Values for long options that do not have single-letter equivalents.  */
 enum
 {
-  DIFF_PROGRAM_OPTION = CHAR_MAX + 1,
+  DIFF_PROGRAM_OPTION = 256,
   HELP_OPTION
 };
 
--- ./src/sdiff.c.orig	Mon Mar 11 09:00:47 2002
+++ ./src/sdiff.c	Wed May  1 12:49:39 2002
@@ -131,7 +131,7 @@
 /* Value for the long option that does not have single-letter equivalents.  */
 enum
 {
-  DIFF_PROGRAM_OPTION = CHAR_MAX + 1,
+  DIFF_PROGRAM_OPTION = 256,
   HELP_OPTION,
   STRIP_TRAILING_CR_OPTION
 };
--- ./src/diff.c.orig	Wed May  1 12:50:30 2002
+++ ./src/diff.c	Wed May  1 12:50:43 2002
@@ -144,7 +144,7 @@
 /* Values for long options that do not have single-letter equivalents.  */
 enum
 {
-  BINARY_OPTION = CHAR_MAX + 1,
+  BINARY_OPTION = 256,
   FROM_FILE_OPTION,
   HELP_OPTION,
   HORIZON_LINES_OPTION,
